EIPS/eip-1.md

```diff
@@ -37,11 +37,31 @@ An EIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## EIP Work Flow
 
+### Shepherding an EIP
+
 Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
 
-:warning: Before you begin, vet your idea, this will save you time. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+Before you begin writing a formal EIP, you should vet your idea. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. Proposing an EIP which has not been properly dilligenced may be grounds for EIP Editors not granting the "Draft" status. It is thus recommended to open a discussion thread on [the Ethereum Magicians forum] to do this, but you can also use [one of the Ethereum Gitter chat rooms], [the Ethereum subreddit] or [the Issues section of this repository]. 
+
+In addition to making sure your idea is original, it will be your role as the champion to socialize your EIP and build community consensus around it. You should try and gauge whether the interest in your EIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the community interest required for a Core EIP will have to be much greater than for an ERC. While "community interest" will not be a factor in deciding whether an EIP gets merged as a draft, insufficient community interest may prevent your EIP from moving forward, especially if it is a Core EIP. 
```

Comment:
"socialize" and "community interest" are very vague.

An EIP should be vetted as technically correct and as bringing value to Ethereum and not to only one interested party/project/developer. This can only be done by editors/developers with expertise in the EIP's topic.

Sure, there should be a call for feedback to the entire community, but EIP marketing (whatever community interest means) should not be more important than having knowledgeable editors/devs vet the idea. EIP marketing is for contentious proposals.

> insufficient community interest may prevent your EIP from moving forward

This sounds like a high-school popularity contest, not the core of a Global OS.

Reply:
Thanks for the comments, @loredanacirstea! 

> "socialize" and "community interest" are very vague.

I agree, but IMO it's still worth highlighting the "soft requirements" of the process, given it's not documented anywhere else. It seems like EIPs do require some social consensus around them before moving forward that is more than just "this is technically sound". 

> Sure, there should be a call for feedback to the entire community, but EIP marketing (whatever community interest means) should not be more important than having knowledgeable editors/devs vet the idea. EIP marketing is for contentious proposals.

I didn't mean to imply that is was. The "commensurate with the work involved" bit is meant to capture this. 

> This sounds like a high-school popularity contest, not the core of a Global OS.

I think this is more about opportunity cost vs. "popularity" per say. If Ethereum is a Global OS, then there needs to be _some_ threshold for what becomes a live upgrade to the protocol. There simply isn't the bandwidth for core devs and EIP editors, in the current process to consider every proposal, especially if the rate at which they come in keeps growing. Having _some_ people who care about actually seeing your EIP go live on Ethereum can act as a filter (and to some extent, already does) for this. 

I'd definitely love to hear suggestions of a better way to (1) phrase this and (2) address the core issue 😄

Reply:
> An EIP should be vetted as technically correct and as bringing value to Ethereum and not to only one interested party/project/developer. This can only be done by editors/developers with expertise in the EIP's topic.

What you're proposing here is a big departure from the current EIP process, both in intent and in execution.

Further, we've been moving towards merging drafts early and often, because it smooths the process, and we gain nothing by long contentious arguments about whether an EIP is "good enough" before merging as a draft. Doing this also results in those discussions getting fragmented - they're on the PR, and if it *does* get merged as a draft, they get lots when discussion moves to the discussions-to URL.

Really, we gain nothing by making it harder to merge a draft; it only makes everyone's lives more difficult.

Reply:
@Arachnid would you be okay with the proposed changes in the PR (not comments) if this sentence was simply removed? It also removes the need to find a real-word-synonym for `dilligenced` 😄  
> Proposing an EIP which has not been properly dilligenced may be grounds for EIP Editors not granting the "Draft" status.

Reply:
@timbeiko ,

> insufficient community interest may prevent your EIP from moving forward, especially if it is a Core EIP
 
How much interest? From how many people? Where do you go to get this community interest and how is it shown? Socializing should not hinder innovation.
If we want more involvement from outside, authors should know who they can ask for an opinion and be able to document this in writing. [This proposal](https://github.com/ethereum/EIPs/pull/1725) is a beginning. Having a list of people/groups who you can ping for feedback opens the process. Because “community interest” is not quantified and there is no list, why add it?
 
As for a text suggestion:
 
"In addition to making sure your idea is original, it will be your role as the author to make your idea clear to reviewers and interested parties, as well as inviting editors, developers and community to give feedback on the aforementioned channels. You should try and gauge whether the interest in your EIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the work required for implementing a Core EIP will be much greater than for an ERC and the EIP will need sufficient interest from the Ethereum client teams. Negative community feedback will be taken into consideration and may prevent your EIP from moving forward."

Reply:
Thanks, @loredanacirstea! I like your suggested paragraph. 

I'm somewhat disinclined to providing an explicit definition for things like "the community", "how much interest", etc. because it is fuzzy by its very nature. On the other hand, it _is_ a part of the process, so I feel like it's best to have it described there, albeit in a fuzzy manner, than not described at all, given the impression that only a technical spec is necessary, while historically that hasn't been the case.

Reply:
@loredanacirstea thanks again for your suggestion. I've added it with a minor modification on the last line in the latest comment.